### DISCUSSION TITLE
I noticed that 22621.3527.65.2 no longer is considered the newest release

### DISCUSSION THREAD

**Discussion by pyrates999** [valinet/ExplorerPatcher#3264] [General]
I noticed that [22621.3527.65.2](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.2_c245b8f) no longer has the latest tag attached to it.  What was the reason for that?  I've already updated to it before it was changed.

**Comment by Amrsatrio** [comment]
Flagged by Microsoft as "a software we don't like". 65.1 will continue to be the latest release as long as new versions keep being flagged. Had to sacrifice the user menu in the 10 start menu fix though...

**Reply by pyrates999** [reply]
understood.
